hera_mc/geo_handling.py

```diff
@@ -144,38 +144,44 @@ def is_in_geo_location(self, station_name):
             station_present = False
         return station_present
 
-    def is_in_connections(self, station_name, active_date=None):
+    def is_in_connections(self, station_name, active_date=None, return_antrev=True):
```

Comment:
If the function is going to stay named `is_in_connections`, the default value of `return_antrev` should be False — otherwise the function name makes it sound like it returns a boolean, but the default behavior is to not do this.

This function is getting a bit Swiss-Army-knifey — it might be simpler to break it into multiple functions, each with more focused semantics.